.github/workflows/ci.yml

```diff
@@ -22,32 +22,32 @@ jobs:
         run: |
           matrix="$(cat <<'EOL'
           [
-            { "name": "amazonlinux-2", "arch": "amd64" },
-            { "name": "centos-7", "arch": "amd64" },
-            { "name": "centos-8", "arch": "amd64" },
-            { "name": "debian-10", "arch": "amd64" },
-            { "name": "debian-11", "arch": "amd64" },
-            { "name": "debian-12", "arch": "amd64" },
-            { "name": "mariner-2", "arch": "amd64" },
-            { "name": "oraclelinux-7", "arch": "amd64" },
-            { "name": "oraclelinux-8", "arch": "amd64" },
-            { "name": "rhel-7", "arch": "amd64" },
-            { "name": "rhel-8", "arch": "amd64" },
-            { "name": "rhel-9", "arch": "amd64" },
-            { "name": "sles-15", "arch": "amd64" },
-            { "name": "ubuntu-20.04", "arch": "amd64" },
-            { "name": "ubuntu-22.04", "arch": "amd64" }
+            { "name": "amazonlinux-2", "arch": "amd64", "tag": "latest" },
+            { "name": "centos-7", "arch": "amd64", "tag": "latest" },
+            { "name": "centos-8", "arch": "amd64", "tag": "latest" },
+            { "name": "debian-10", "arch": "amd64", "tag": "latest" },
+            { "name": "debian-11", "arch": "amd64", "tag": "latest" },
+            { "name": "debian-12", "arch": "amd64", "tag": "latest" },
+            { "name": "mariner-2", "arch": "amd64", "tag": "latest" },
```

Comment:
>{ "name": "mariner-2", "arch": "amd64", "tag": "latest" }, [](http://example.com/codeflow?start=12&length=58)

Rename to Azure Linux 2 and move to the top of the list, pls

Reply:
Thats part of a larger change, those targets must match the Dockerfiles, which is set to mariner sadly. Ill have a follow-up PR for when native arm64 targets come with this fixed.